CHANGELOG.md

```diff
@@ -1,5 +1,17 @@
 # Changelog
 
+## 24.1.0
```

Comment:
The next version is 23.10.2, unless you specifically want to add it to the next quarterly